src/Sarif/Schemata/sarif-schema.json

```diff
@@ -2315,6 +2377,97 @@
             "required": [ "name" ]
         },
 
+        "toolComponentTranslation": {
+            "description": "TBD.",
```

Comment:
>TBD [](start = 28, length = 3)

"Provides localized message strings for a single tool component in a single language."